srml/executive/src/mock.rs

```diff
@@ -0,0 +1,125 @@
+// Copyright 2019 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Test utilities
+
+#![cfg(test)]
+
+use super::*;
+use runtime_io;
+use substrate_primitives::{Blake2Hasher};
+use srml_support::{impl_outer_origin, impl_outer_event, impl_outer_dispatch, parameter_types};
+use primitives::traits::{IdentityLookup, BlakeTwo256};
+use primitives::testing::{Header, Block};
+use system;
+pub use balances::Call as balancesCall;
+pub use system::Call as systemCall;
+use node_runtime::impls::WeightMultiplierUpdateHandler;
```

Comment:
Invalid. The point of this is to test *executive*. The test runtime exists as a minimal runtime designed to facilitate that in a controlled fashion. This should not be testing the fully-featured substrate-node runtime, nor should it be pulling in anything from that set of crates.

Reply:
Will take this into account but do note that this PR will probably have to be closed and a portion of it need to be ported to into your PR as a signed extension.